WEBAPI.Infrastructure/DB/ApplicationDbContext.cs

```diff
@@ -0,0 +1,236 @@
+﻿using Microsoft.Data.SqlClient;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using WEBAPI.Domain.Entities;
+using BCrypt.Net;
+
+namespace WEBAPI.Infrastructure.DB
+{
+    public class ApplicationDbContext
+
+    {
+        public string ConnectionString { get; set; }
+
+        public ApplicationDbContext(string connectionstr)
+        {
+            ConnectionString = connectionstr;
+
+        }
+
+
+        public List<User> GetUsers()
+        {
+            List<User> _userDetails = new List<User>();
+            string connectionString = ConnectionString;
+
+            using (SqlConnection connection = new SqlConnection(connectionString))
+            {
+                connection.Open();
+
+                string sql = "SELECT * FROM Users"; // Replace YourTableName with your actual table name
```

Comment:
it's good to use store procedure or use abstraction

Reply:
Resolved.